js/externals.js

```diff
@@ -1,94 +1,139 @@
-  'components/Accordion/AccordionBlock': 'AccordionBlock',
-  'components/Button/Button': 'Button',
-  'components/Button/BackButton': 'BackButton',
-  'components/Breadcrumb/Breadcrumb': 'Breadcrumb',
-  'components/FormAction/FormAction': 'FormAction',
-  'components/FormBuilder/FormBuilder': 'FormBuilder',
-  'components/FormBuilderModal/FormBuilderModal': 'FormBuilderModal',
-  'components/FileStatusIcon/FileStatusIcon': 'FileStatusIcon',
-  'components/FieldHolder/FieldHolder': 'FieldHolder',
-  'components/GridField/GridField': 'GridField',
-  'components/Toolbar/Toolbar': 'Toolbar',
-  'components/LiteralField/LiteralField': 'LiteralField',
-  'components/Preview/Preview': 'Preview',
-  'components/ListGroup/ListGroup': 'ListGroup',
-  'components/ListGroup/ListGroupItem': 'ListGroupItem',
-  'components/Loading/Loading': 'Loading',
-  'components/FormAlert/FormAlert': 'FormAlert',
-  'components/Badge/Badge': 'Badge',
-  'components/VersionedBadge/VersionedBadge': 'VersionedBadge',
-  'components/TreeDropdownField/TreeDropdownField': 'TreeDropdownField',
-  'components/Focusedzone/Focusedzone': 'Focusedzone',
-  'components/ViewModeToggle/ViewModeToggle': 'ViewModeToggle',
-  'components/ResizeAware/ResizeAware': 'ResizeAware',
-  'components/Tag/Tag': 'Tag',
-  'components/Tag/TagList': 'TagList',
-  'components/Tag/CompactTagList': 'CompactTagList',
-  'components/Search/Search': 'Search',
-  'components/Search/SearchToggle': 'SearchToggle',
-  'containers/FormBuilderLoader/FormBuilderLoader': 'FormBuilderLoader',
-  'containers/InsertMediaModal/InsertMediaModal': 'InsertMediaModal',
-  'containers/InsertLinkModal/InsertLinkModal': 'InsertLinkModal',
-  'containers/InsertLinkModal/fileSchemaModalHandler': 'FileSchemaModalHandler',
-  'state/breadcrumbs/BreadcrumbsActions': 'BreadcrumbsActions',
-  'state/schema/SchemaActions': 'SchemaActions',
-  'state/toasts/ToastsActions': 'ToastsActions',
-  'state/records/RecordsActions': 'RecordsActions',
-  'state/records/RecordsActionTypes': 'RecordsActionTypes',
-  'state/tabs/TabsActions': 'TabsActions',
-  'state/viewMode/ViewModeActions': 'ViewModeActions',
-  'lib/DataFormat': 'DataFormat',
-  'lib/Backend': 'Backend',
-  'lib/getFormState': 'getFormState',
-  'lib/ReactRouteRegister': 'ReactRouteRegister',
-  'lib/ReducerRegister': 'ReducerRegister',
-  'lib/SilverStripeComponent': 'SilverStripeComponent',
-  'lib/formatWrittenNumber': 'formatWrittenNumber',
-  'lib/Router': 'Router',
-  'lib/schemaFieldValues': 'schemaFieldValues',
-  'lib/Config': 'Config',
-  'lib/Injector': 'Injector',
-  'lib/reduxFieldReducer': 'reduxFieldReducer',
-  'lib/TinyMCEActionRegistrar': 'TinyMCEActionRegistrar',
-  'lib/ShortcodeSerialiser': 'ShortcodeSerialiser',
-  'lib/withDragDropContext': 'withDragDropContext',
-});
+  // Don't include the exports provided by the current module
+  if (module && exports[module] !== undefined) {
+    delete exports[module];
+  }
```

Comment:
The way exports behaves seems to have changed - if the file which is _exporting_ these also has them as _external_ it won't actually include them in the bundle, so they don't get exported and there are errors all over the place.